gamemodes/terrortown/gamemode/shared/hud_elements/base_elements/pure_skin_element.lua

```diff
@@ -32,14 +34,46 @@ if CLIENT then
 		self:DrawLines(x, y, w, h, c.a)
 
 		-- draw text
-		self:ShadowedText(t or "", "PureSkinBar", x + 14, y + 1, COLOR_WHITE, TEXT_ALIGN_LEFT, TEXT_ALIGN_LEFT)
+		if t then
+			self:AdvancedText(t, "PureSkinBar", x + 14, y + 1, COLOR_WHITE, TEXT_ALIGN_LEFT, TEXT_ALIGN_LEFT, true, s)
+		end
 	end
 
 	function HUDELEMENT:ShadowedText(text, font, x, y, color, xalign, yalign)
 		draw.SimpleText(text, font, x + 2, y + 2, shadowColor, xalign, yalign)
 		draw.SimpleText(text, font, x + 1, y + 1, shadowColor, xalign, yalign)
 		draw.SimpleText(text, font, x, y, color, xalign, yalign)
 	end
+	
+	function HUDELEMENT:AdvancedText(text, font, x, y, color, xalign, yalign, shadow, scale)
+		local mat
+		if isvector(scale) or scale ~= 1.0 then
+			mat = Matrix()
+			mat:Translate(Vector(x, y))
+			
+			mat:Scale(isvector(scale) and scale or Vector(scale, scale, scale))
+			mat:Translate(-Vector(x, y))
+
+			render.PushFilterMag(TEXFILTER.ANISOTROPIC)
+			render.PushFilterMin(TEXFILTER.ANISOTROPIC)
+
+			cam.PushModelMatrix(mat)
+		end
+		
+		if shadow then
+			self:ShadowedText(text, font, x, y, color, xalign, yalign)
+		
```

Comment:
I know it's not as much important, but pls give a little bit more attention to useless empty lines :) Just to have a clean code. Some empty lines are important and useful, but there is never a useful empty line in front of an structure-element

Reply:
Yeah I should pay more attention when I'm deleting lines of code ^^